packages/sdk/.speakeasy/workflow.lock

```diff
@@ -1,9 +1,9 @@
-speakeasyVersion: 1.396.3
+speakeasyVersion: 1.396.4
 sources: {}
 targets:
     vercel:
         source: vercel-OAS
-        outLocation: /home/disintegrator/github.com/vercel/vercel/packages/sdk
+        outLocation: /Users/ismael/Code/vercel/speakeasy/vercel/packages/sdk
```

Comment:
Though it looks surprising it only indicates the location where generation was run. It will be replaced with the location on the GitHub runner on the next automated generation. I’ll follow up with our team if we need to keep this detail in our lock file but it’s not a blocker from our side.

Reply:
Spoke with the team and we’ve just pushed an update to remove this line from our lockfile. It will be in the upcoming release of our CLI in the next hour or so.